platforms/documentation/docs/src/docs/userguide/authoring-builds/structuring/multi_project_builds.adoc

```diff
@@ -259,3 +196,98 @@ All letters are lowercase, and words are separated with a dash (`-`) character.
+
+[[declaring_dependencies_between_subprojects]]
+== Declaring dependencies between Subprojects
+
+What if one subproject depends on another?
+What if one subproject depends on the artifact produced by another?
+
+image::structuring-builds-7.png[]
+
+This is a common use case in multi-project builds.
+Gradle supports this scenario with <<declaring_dependencies_basics.adoc#sec:project-dependencies,project dependencies>>.
+
+[[sec:project_jar_dependencies]]
+=== Depending on another Project
+
+Consider a multi-project build with the following layout:
+
+[.multi-language-sample]
+=====
+[source, kotlin]
+----
+.
+├── api
+│   ├── src
+│   │   └──...
+│   └── build.gradle.kts
+├── services
+│   └── person-service
+│       ├── src
+│       │   └──...
+│       └── build.gradle.kts
+├── shared
+│   ├── src
+│   │   └──...
+│   └── build.gradle.kts
+└── settings.gradle.kts
+----
+=====
+[.multi-language-sample]
+=====
+[source, groovy]
+----
+.
+├── api
+│   ├── src
+│   │   └──...
+│   └── build.gradle
+├── services
+│   └── person-service
+│       ├── src
+│       │   └──...
+│       └── build.gradle
+├── shared
+│   ├── src
+│   │   └──...
+│   └── build.gradle
+└── settings.gradle
+----
+=====
+
+In this example:
+
+1. `person-service` depends on both `api` and `shared`
+2. `api` depends on `shared`
+
+You declare these relationships using the project path, which uses colons (`:`) to indicate nesting.
+For example:
+
+- `:shared` refers to the `shared` subproject
+- `services:person-service` refers to a nested subproject
+
+[[javadependencies_2]]
+====
+include::sample[dir="snippets/multiproject/basic-dependencies/kotlin",files="settings.gradle.kts[];shared/build.gradle.kts[];api/build.gradle.kts[];services/person-service/build.gradle.kts[]"]
+include::sample[dir="snippets/multiproject/basic-dependencies/groovy",files="settings.gradle[];shared/build.gradle[];api/build.gradle[];services/person-service/build.gradle[]"]
+====
+
+For more details on project paths, consult the DSL documentation for link:{groovyDslPath}++/org.gradle.api.initialization.Settings.html#org.gradle.api.initialization.Settings:include(java.lang.String[])++[Settings.include(String...)].
+
+A project dependency affects both the build order and classpath:
+
+- The required project will be built first.
+- Its compiled classes and transitive dependencies are added to the consuming project’s classpath.
+
+For example, running `./gradlew :api:compileJava` will first build `shared`, then `api`.
+
+[[sec:depending_on_output_of_another_project]]
+=== Depending on Artifacts produced by another Project
+
+Sometimes, you only need the output of a specific task from another project—not the entire project itself.
+
+While you can create task-to-task dependencies between projects, Gradle discourages this because it creates tight coupling between tasks.
+
+Instead, use outgoing artifacts to expose a task’s output and model it as a dependency.
```

Comment:
This is a really important point. Is there an example of how to do this correctly somewhere else in the docs?